### PR TITLE
Master spreadsheet dev pro

### PR DESCRIPTION
**Pull request by pro-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/js/chrome/abstract_action.js [hunk 1]
```diff
@@ -27,6 +27,12 @@ var AbstractAction = Widget.extend(ActionMixin, {
      */
     hasControlPanel: false,
 
+    /**
+     * Owl component to render as the control panel. The component
+     * should extend the regular control panel component.
+     */
+    controlPanelComponent: ControlPanel,
```
Comment (VincentSchippefilt) [inline addons/web/static/src/js/chrome/abstract_action.js:34]: this commit needs a test and can be merged in master without waiting for spreadsheet

Comment (VincentSchippefilt) [reply addons/web/static/src/js/chrome/abstract_action.js:34]: test should create an action with a control panel that is not the default control panel

**Comment by VincentSchippefilt** [comment]
in fact, this entire PR can be merged in master, provided that tests are written to engrave the behavior that we want to have for spreadsheet

**Comment by pro-odoo** [comment]
https://github.com/odoo/odoo/pull/52807

**Pull request closed without merging**